Packs/GenianNAC/Integrations/GenianNAC/GenianNAC.py

```diff
@@ -187,9 +187,9 @@ def main():
         if demisto.command() == "test-module":
             get_ip_nodeid('8.8.8.8')
             demisto.results('ok')
-        elif demisto.command() == 'genians-assign-ip-tag':

```

Comment:
Note that when you update commands name and outputs is breaking backward compatibility.

Reply:
I will revert it back.